### DISCUSSION TITLE
DAI 2023-2024 - Practical work 2 - Sarah Jallon + Jonas Troeltsch

### DISCUSSION THREAD

**Discussion by sarsurgithub** [heig-vd-dai-course/heig-vd-dai-course#146] [Show and tell]
We will create a protocol and an app to play mastermind 

[Repository](https://github.com/sarsurgithub/DAI-PW2-Mastermind)

**Comment by jonas-tr** [comment]
I'm available

**Comment by jonas-tr** [comment]
last commit: 65a71a4a9e919b818c07a170e071003b0ab0031f
